media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp

```diff
@@ -506,9 +506,23 @@ VAStatus DdiEncodeHevc::ParsePicParams(
 
     PCODEC_HEVC_ENCODE_SEQUENCE_PARAMS hevcSeqParams = (PCODEC_HEVC_ENCODE_SEQUENCE_PARAMS)((uint8_t *)m_encodeCtx->pSeqParams);
 
+    DDI_MEDIA_SURFACE *surface;
     if(picParams->decoded_curr_pic.picture_id != VA_INVALID_SURFACE)
     {
-        DDI_CHK_RET(RegisterRTSurfaces(&(m_encodeCtx->RTtbl), DdiMedia_GetSurfaceFromVASurfaceID(mediaCtx, picParams->decoded_curr_pic.picture_id)), "RegisterRTSurfaces failed!");
+        surface = DdiMedia_GetSurfaceFromVASurfaceID(mediaCtx, picParams->decoded_curr_pic.picture_id);
+
+        if(m_encodeCtx->vaEntrypoint==VAEntrypointEncSliceLP
```

Comment:
Should allow VAEntrypointEncSlice as well for VMEPAK supported format as well, at least for Y210.
Otherwise 422 10 BIT encoding still fails for VMEPAK path.

Reply:
thanks, yes, it is for both VME path and VDEncPath